include/deal.II/lac/petsc_vector.h

```diff
@@ -275,20 +275,16 @@ namespace PETScWrappers
       clear() override;
 
       /**
-       * Copy the given vector. Resize the present vector if necessary. Also
-       * take over the MPI communicator of @p v.
+       * Explicitly use the base class operator= functions.
        */
-      Vector &
-      operator=(const Vector &v);
+      using VectorBase::operator=;
```

Comment:
Why is this correct if you're providing your own copy operator below?

Reply:
I think in order to reason about this better, I'd rather you duplicate the base class's declaration here and in the implementation just call the corresponding function of the base class.

Reply:
More specifically, you're importing *all* `operator=` overloads here. I think I'd favor keeping the overload you removed below.